### PR TITLE
min proj file and make work on windows.

### PR DESCRIPTION
**Pull request by yonderblue**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,13 +30,23 @@ queue.Ack(message);
...
+For linux run:
 
 ```bash
 xbuild DominionEnterprises.Mongo/DominionEnterprises.Mongo.csproj
 ```
 
-and use the resulting `DominionEnterprises.Mongo/bin/Debug/DominionEnterprises.Mongo.dll` in your project.
+For windows run:
+
+```
```
Comment (nubs) [inline README.md]: I'd recommend throwing some syntax highlighting for these too as the font size ends up being different between this and the bash example.  Either `bash` or `bat` could probably be used for these.

Comment (yonderblue) [inline README.md]: ya it wouldnt be bash but i couldnt find the windows shell syntax in there list. but ill stick bash on it

Comment (nubs) [inline README.md]: `bat` is available if you'd rather that.

**Comment by yonderblue** [comment]
@nubs fixed

**Pull request merged**